setup.py

```diff
@@ -1,8 +1,24 @@
 from os import path
 from setuptools import setup, find_packages
+from setuptools.command.install import install
+from subprocess import check_call, CalledProcessError
 
 from apertium_apy import apy
 
+
+class InstallHelper(install):
+    def run(self):
+        try:
+            check_call(['make', 'langNames.db'])
+        except CalledProcessError:
+            pass
+        install.run(self)
```

Comment:
`super().run()` is preferred for its DRYness and conciseness.

Reply:
But the instance is passed in so it's not a class hierarchy, so how can super() be relevant here? I don't actually know Python...make whatever code changes you want.

Reply:
The bad naming convention is the real culprit here. `install` is a class. `class InstallHelper(install):` means here's a class `InstallHelper` that extends `install`. I can make the changes if you'd rather not.